src/gui/shellwidget/shellwidget.cpp

```diff
@@ -239,13 +239,13 @@ void ShellWidget::paintUnderline(
 static QPainterPath GetUndercurlPath(QRect cellRect) noexcept
 {
 	const QLine underline{ GetUnderline(cellRect) };
-	const QPoint& start{ underline.p1() };
-	const QPoint& end{ underline.p2() };
+	const QPointF& start{ underline.p1() };
```

Comment:
The `&` is no longer necessary.

`QPoint` to `QPointF` conversion, so a new object is created either way.

Reply:
Actually there is no floating-point operations and the cause was not related to rounding, I'll restore the original code around here.